src/playbacks/html5_audio/html5_audio.js

```diff
@@ -61,11 +61,11 @@ export default class HTML5Audio extends Playback {
   }
 
   getPlaybackType() {
-    return [0, undefined, Infinity].indexOf(this.el.duration) >= 0 ? 'live' : 'aod'
+    return [0, undefined, Infinity].indexOf(this.el.duration) >= 0 ? Playback.LIVE : Playback.AOD
   }
 
   stalled() {
-    if (this.getPlaybackType() === 'vod' && this.el.readyState < this.el.HAVE_FUTURE_DATA) {
+    if (this.getPlaybackType() === Playback.VOD && this.el.readyState < this.el.HAVE_FUTURE_DATA) {
```

Comment:
I'm sure that we're making a mistake here, we should be checking for `AOD`. However, it seems that @dayvson is working on #466 so we should remove the entire html5_audio soon